static/js/billing/billing.js

```diff
@@ -108,7 +108,7 @@ $(function () {
                     url: "/json/billing/upgrade",
                     data: {
                         stripe_token: JSON.stringify(stripe_token.id),
-                        csrfmiddlewaretoken: $("#autopay-form input[name='csrf']").val(),
+                        csrfmiddlewaretoken: get_form_input("autopay", "crf", false),
```

Comment:
@showell Yeah. I have removed passing `csrfmiddlewaretoken` as data altogether since our csrf.js automatically set the CSRFToken header to the value of `input[name="csrfmiddlewaretoken"]`.